www-misc/profile-sync-daemon/profile-sync-daemon-6.35.ebuild

```diff
@@ -11,15 +11,17 @@ KEYWORDS="~amd64 ~x86"
 
 LICENSE="MIT"
 SLOT="0"
-IUSE=""
+IUSE="systemd"
 
 RDEPEND="
 	app-shells/bash
 	net-misc/rsync[xattr]
-	sys-apps/systemd"
+	systemd? ( sys-apps/systemd )"
 
 src_install() {
-	emake DESTDIR="${D}" COMPRESS_MAN=0 install
+	emake DESTDIR="${D}" COMPRESS_MAN=0 install-bin
+	emake DESTDIR="${D}" COMPRESS_MAN=0 install-man
+		"$(usex systemd "install-systemd")"
```

Comment:
so the only difference is installing systemd unit files?

Reply:
Not exactly. I made the systemd dependency optional by adding a USE flag for it. Beforehand it was not possible to install profile-sync-daemon on non-systemd systems. And since the makefile has a separate target for the systemd unit files, I figured those should also be only installed, if the USE=systemd is set.

Reply:
[Small files should be installed unconditionally.](https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301) So we could remove the dependency on systemd altogether and not need this `usex` line. It would be nice if you could add an example cron file or OpenRC service script.